storage/innobase/include/dict0dict.ic

```diff
@@ -300,6 +336,18 @@ dict_index_is_clust(
 	return(index->type & DICT_CLUSTERED);
 }
 
+/********************************************************************//**
+Check whether the index is the clustered index of instant table.
+@return nonzero for clustered index of instant table, zero for other indexes */
+UNIV_INLINE
+ulint
+dict_index_is_clust_instant(
+/*================*/
+	const dict_index_t*	index)	/*!< in: index */
+{
+	return dict_index_is_clust(index) && dict_table_is_instant(index->table);
+}
+
```

Comment:
I would prefer to define this as a method: `bool dict_index_t::is_instant() const`, preferrably inline in the .h file.
(The `clust` is redundant; instant ADD COLUMN will not be possible for any secondary indexes.)
Similarly, the table-level method should be `bool dict_table_t::is_instant() const`.